README.md

```diff
@@ -0,0 +1,29 @@
+# SciGuides
+
+Welcome to SciGuides, a repository made by FRC Team 1155 and 2265 to gather all of our experience and knowledge for public and internal use.
+
+## Usage
```

Comment:
maybe add specifics about the sections? Or just like link to the projects readme (speaking of, there should probably be more of a projects readme)